tests/ignite/metrics/test_ssim.py

```diff
@@ -123,6 +123,25 @@ def test_ssim_variable_batchsize(available_device):
     assert np.allclose(out, expected)
 
 
+def test_cuda_ssim_dtypes(available_device):
+    # Checks https://github.com/pytorch/ignite/pull/3034
+    # this test should not be run on CPU
```

Comment:
Why this test should not run on CPU ?

Reply:
Because the F.pad operation does not exist for Half type on CPU

Reply:
I could check on CPU if the code effectively gives a RuntimeError for Half and Double type.

Reply:
I think float32 and double should be supported on cpu, so let's run on it

Reply:
You are right ! I just tested. Thank you for your help :)

Reply:
Cool, sounds good, can you please also remove above comment as it is less relevant:
```
    # Checks https://github.com/pytorch/ignite/pull/3034
    # this test should not be run on CPU
```